Avoid an error caused by modifying dict while iterating it.

Author: Alexander Grund (TU Dresden)

diff --git a/tools/flight_recorder/components/types.py b/tools/flight_recorder/components/types.py
index 20e093688ba..98192aeb92c 100644
--- a/tools/flight_recorder/components/types.py
+++ b/tools/flight_recorder/components/types.py
@@ -164,7 +164,7 @@ class Database(NamedTuple):
 # TODO: We need to add a schema for the following
 types = [
     TypeInfo.from_type(t)  # type: ignore[type-var]
-    for t in globals().values()
+    for t in list(globals().values())
     if (
         isinstance(t, type)
         and issubclass(t, tuple)
